tasks/store-publish/task.json

```diff
@@ -117,6 +117,15 @@
             "required": false,
             "helpMarkDown": "Paths to any additional packages required by this application (one path per line). Minimatch pattern is supported."
         },
+        {
+            "name": "commitSubmission",
+            "type": "boolean",
+            "label": "Commit Submission",
+            "defaultValue": true,
+            "groupName": "advanced",
+            "required": true,
+            "helpMarkDown": "Commit the submission after creating it in Dev Center. Not submitting allows Marketing teams to manually review the submission details before committing."
+        },
         {
             "name": "skipPolling",
```

Comment:
You can add a visual condition to this element so it doesn't show up if the `commitSubmission` is false. See examples in other inputs. The code that overrides this options is still needed.